firestore/tests/unit/v1/test_field_path.py

```diff
@@ -493,3 +493,8 @@ def test_lineage_nested(self):
         field_path = self._make_one("a", "b", "c")
         expected = set([self._make_one("a"), self._make_one("a", "b")])
         self.assertEqual(field_path.lineage(), expected)
+
+    def test_document_id(self):
+        parts = "__name__"
+        field_path = self._make_one(parts)
+        self.assertEqual(field_path.document_id(), "__name__")
```

Comment:
can we do
`self.assertEqual(field_path.document_id(), parts)`
instead of
`self.assertEqual(field_path.document_id(), "__name__")`

Reply:
I'd also change "\_\_name\_\_" to something else, like "test_id" or whatever, to avoid possible confusion with `__name__`

Reply:
@mf2199 May be we can't change "name" to something else because in back-end may be they are replacing __name__ with id and we also have to match with nodejs , so they are using __name__ in other languages.